testsuite/tests/tool-ocamlc-compat32/Makefile

```diff
@@ -0,0 +1,17 @@
+BASEDIR=../..
+
+compile:
+	printf " ... testing -compat-32"
+	@$(OCAMLC) -compat-32 -c a.ml 2>&1 | xargs echo ${} > test.result  
```

Comment:
On the basis that tests should be as simple as possible, why is the output piped to `xargs`?